docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -88,7 +88,7 @@ You can attach tags to the application manually, instead of enabling automated t
 +
 [NOTE]
 ====
-Analysis engines use standard rules for a comprehensive set of migration targets, but if the target is not included or is a customized framework, custom rules can be added. Custom rules files are validated.
+Analysis engines use standard rules for a comprehensive set of migration targets, but if the target is not included or is a customized framework, custom rules can be added. Only manually uploaded custom rule files are validated.
```

Comment:
@emarcusRH the word "only" sounds redundant. 
Can you please make it:  `Manually uploaded custom rule files are validated`?

Reply:
That depends on what is the actual case here - if you can only validated manually uploaded files, then "only" is needed here. What is the functionality here? is there an automatic upload option, or files that were already uploaded  before hand?

Reply:
There are 2 options to include rule files in analysis:
- by given a git source repository where rule files are located (no validation)
- by manually upload files from local source (files are validated against schema file)

I think that the word "only" is missing context here and redundant because the 2 options are not mentioned in this specific note. But I agree to leave it as it is if you think it is correct. Thanks